splash/tests/test_execute.py

```diff
@@ -332,6 +332,67 @@ def test_pcall_wrong_keyword_arguments(self):
         self.assertEqual(data["ok"], False)
 
 
+class EnableDisableJSTest(BaseLuaRenderTest):
+
+    def test_disablejs(self):
+        resp = self.request_lua("""
+        function main(splash)
+            splash.js_enabled = false
+            splash:go(splash.args.url)
+            local html = splash:html()
+            return html
+        end
+        """, {
+            'url': self.mockurl('jsrender'),
+        })
+        self.assertStatusCode(resp, 200)
+        self.assertIn(u'Before', resp.text)
+
+    def test_enablejs(self):
+        resp = self.request_lua("""
+        function main(splash)
+            splash.js_enabled = true
+            splash:go(splash.args.url)
+            local html = splash:html()
+            return html
+        end
+        """, {
+            'url': self.mockurl('jsrender'),
+        })
+        self.assertStatusCode(resp, 200)
+        self.assertIn(u'After', resp.text)
```

Comment:
@sunu , I think you should **assertNotIn(u'Before', resp.text)** rather than **assertIn(u'After', resp.text)**, because 'After' will be present in the html's script tag anyways.